src/qibolab/__init__.py

```diff
@@ -79,3 +79,47 @@ def execute_qasm(circuit: str, platform, runcard=None, initial_state=None, nshot
     return QibolabBackend(platform, runcard).execute_circuit(
         circuit, initial_state=initial_state, nshots=nshots
     )
+
+
+def get_available_platforms() -> list[str]:
+    """Returns the platforms found in the $QIBOLAB_PLATFORMS directory."""
+    return [
+        d.name
+        for d in get_platforms_path().iterdir()
+        if d.is_dir() and not (d.name.startswith("_") or d.name.startswith("."))
```

Comment:
This is fine, just looks a bit hardcoded to the format we are using in qibolab_platforms_qrc repo. Maybe there is a more robust way to tell if a directory is a platform, for example by checking if it contains `platform.py` file. 
I think now if I create a random empty folder, whose name doesn't start with `_` or `.`, this will list it as a platform, even though it's not.

Reply:
Well, I just proposed in #849 to remove `platform.py` in favor of `__init__.py`...
But:

1. it was just a proposal, and the current state of the art (which will remain for another while) is a `platform.py` file (in case, this is a change suitable for 0.2)
2. we could add some other marker, or decide that the proper way to check it is to import the corresponding package, and check for a variable marker